src/Panel/TwigPanel.php

```diff
@@ -23,6 +23,11 @@
 
 class TwigPanel extends DebugPanel
 {
+    /**
+     * @var string[]
+     */
+    protected static $extensions = ['.twig'];
```

Comment:
It's better to use the `TwigView::EXT` here instead of hardcoded string.

Reply:
This should be set by TwigView and can be empty by default.